service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -26,25 +26,26 @@ For more information about upgrading Operators, refer to the xref:../../operator
 endif::[]
 
 [id="upgrading-control-plane"]
-== Upgrading the Service Mesh control plane
+== Upgrading the control plane
 
 You must manually update the control plane for minor and major releases.
 
+While you can deploy multiple versions of the control plane in the same cluster, {SMProductName} does not support canary upgrades of the service mesh.  That is, you can have different SCMP resources with different values for `spec.version`, but they cannot be managing the same mesh.
+
 include::modules/ossm-upgrade-21-22-changes.adoc[leveloffset=+2]
 
 For more information about migrating your extensions, refer to xref:../../service_mesh/v2x/ossm-extensions.adoc#ossm-extensions-migration-overview_ossm-extensions[Migrating from ServiceMeshExtension to WasmPlugin resources].
 
-include::modules/ossm-upgrading-from-21-to-22.adoc[leveloffset=+2]
 
 include::modules/ossm-upgrade-20-21-changes.adoc[leveloffset=+2]
 
-include::modules/ossm-upgrading-from-20-to-21.adoc[leveloffset=+2]
+include::modules/ossm-upgrading-smcp.adoc[leveloffset=+2]
 
-include::modules/ossm-migrating-to-20.adoc[leveloffset=+1]
+include::modules/ossm-migrating-to-20.adoc[leveloffset=+2]
 
 [id="upgrading-data-plane"]
 == Upgrading the data plane
 
-You must restart your application pods and workloads to apply updates to the Envoy proxy or changes to proxy configuration.
+Your data plane will still function after you have upgraded the control plane.  But in order to apply updates to the Envoy proxy and any changes to proxy configuration, you must restart your application pods and workloads.
```

Comment:
```suggestion
Your data plane will still function after you have upgraded the control plane.  But to apply updates to the Envoy proxy and any changes to the proxy configuration, you must restart your application pods and workloads.
```

Reply:
See ISG guidelines for 'in order to'. https://www.ibm.com/docs/en/ibm-style?topic=word-usage